Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM fedora:27
+ARG GIT_REPO=https://github.com/Manageiq/integration_tests
+ARG GIT_BRANCH=master
+
+# this can be dropped after the python3 switch
+RUN dnf update -y && dnf install -y python2 'dnf-command(debuginfo-install)'
+
+WORKDIR /integration_tests
+COPY cfme/scripting/ cfme/scripting/
+COPY cfme/__init__.py cfme/__init__.py
+COPY requirements requirements
+COPY setup.py setup.py
+RUN mkdir conf
+RUN python -m cfme.scripting.quickstart --mk-virtualenv /cfme_venf
+
+
+
+RUN dnf install -y git
+RUN git config --global user.name DockerCfme
```

Comment:
Do we need to do anything more to blow away all the git config? Like rm .git/config? Or will this do?

Reply:
this just ensures git works